doc/architectures.md

```diff
@@ -1,26 +1,12 @@
+(architectures)=
 # Architectures
 
-## Introduction
+LXD can run on just about any architecture that is supported by the Linux kernel and by Go.
 
-LXD just like LXC can run on just about any architecture that's
-supported by the Linux kernel and by Go.
+Some objects in LXD are tied to an architecture, for example, the instances, instance snapshots and images.
```

Comment:
I wonder, did we explicitly agree on "objects" term here or is it a carry over from the past?
Not a problem, but I would like to know if this is the term we should be using, as I've tended to use "entities" in the past to avoid confusion with "objects" in the programming sense (given that we sometimes talk about LXD with respect to API development, where the term "object" may be confused for the type definition used in the API to represent the associated entity in LXD).

Reply:
Anyway not a blocker for this PR, just something I thought might be worth discussing/agreeing terminology.

Reply:
This is a carry-over from the existing documentation - but it definitely makes sense to change it, I agree. :)